config.py

```diff
@@ -1,41 +0,0 @@
-import os
```

Comment:
This is not the final content of this file, is it?

Reply:
That's strange.  `import os` was not the content of that file when I posted that review comment.  My comment stands, though.

Reply:
The contents of the config files are subject to change.  I'm just trying to get this going in Docker / OpenShift for the moment.